lib/src/main/java/com/shopify/checkoutsheetkit/CheckoutWebView.kt

```diff
@@ -138,6 +138,8 @@ internal class CheckoutWebView(context: Context, attributeSet: AttributeSet? = n
 
         override fun onPageFinished(view: WebView, url: String) {
             super.onPageFinished(view, url)
+            if (view.contentHeight == 0) return
```

Comment:
what exactly is this checking? 🤔

Reply:
If you look at the URLs in the description:

1. https://shop1.shopify.checkout-web-fvy7.daniel-kift.eu.spin.dev/cart/c/c1-cee737be76e498a6eefa107843517818?key=0f92b50b0375b1c8224d0950e0bb5210
2. https://shop1.shopify.checkout-web-fvy7.daniel-kift.eu.spin.dev/checkouts/cn/c1-cee737be76e498a6eefa107843517818?locale=en-CA&skip_shop_pay=prompt_allowed

The first looks like the cart -> checkout redirect to me, which would have no content.

We shouldn't treat that as a complete load.  We could also potentially use a URL white/blacklist, but URLs are changeable

Reply:
Could you check the response status for a 302 instead?

Reply:
I found it strange yeah

Copying a URL into chrome and checking the network tab, I see these though, which match up:
```
Request URL: https://shop1.shopify.checkout-web-fvy7.daniel-kift.eu.spin.dev/cart/c/c1-e42cf7c4a06bb118769277249677d426?key=f00f2b671b20f8fa4c5af55b1430067f
Request Method: GET
Status Code: 302 Found
```

```
Request URL: https://shop1.shopify.checkout-web-fvy7.daniel-kift.eu.spin.dev/checkouts/cn/c1-e42cf7c4a06bb118769277249677d426
Request Method: GET
Status Code: 200 OK
```

Reply:
Is it not possible to extract the status code from the webview somehow?